README.md

```diff
@@ -12,8 +12,27 @@ To use Vercel Next.js, you need the following tools:
 
 - Node.js - [Install Node.js 14](https://nodejs.org/en/), including the npm package management tool. [NVM](https://github.com/nvm-sh/nvm) is recommended.
 
-1. Copy `.env.example` to `.env` and add values below, then source to shell.
-2. `npm run dev` - to run the Next.js app locally
+1. Clone the repo and change directory
+   ```
+   git clone git@github.com:twos-complement/level-protocol-website.git && cd level-protocol-website
+   ```
+2. Use nvm to use the correct version of node
+   ```
+   nvm use
+   ```
+3. Install node modules. Your package-lock.json should not change, if it did you are probably not using the correct node version. (See step 2)
+   ```
+   npm install
+   ```
+4. Use development environment variables
+   ```
+   cp .env.development .env && source .env
```

Comment:
@itsprayy — This could probably be: `cp .env.development .env.local` and you might not have to source? nextjs should support that approach. (Good call out!)

Reply:
Fixed